src/iperf_util.c

```diff
@@ -105,7 +105,7 @@ void fill_with_repeating_pattern(void *out, size_t outsize)
  * Generate and return a cookie string
  *
  * Iperf uses this function to create test "cookies" which
- * server as unique test identifiers. These cookies are also
+ * serves as unique test identifiers. These cookies are also
```

Comment:
If we're going to fix that word, it should be just "serve" because the word "cookies" is plural.